src/Form/Form.js

```diff
@@ -240,7 +240,8 @@ class Form {
      */
     getValuesReadyForStorage() {
         const formValues = this.getRawValues();
-        return prepareFormValuesForStorage(formValues);
+        const hasOnlyOneCredential = this.inputs.credentials.size === 1;
+        return prepareFormValuesForStorage(formValues, hasOnlyOneCredential);
```

Comment:
Feels like `hasOnlyOneCredential` belongs inside `prepareFormValuesForStorage`, rather than here. Any reason not to have it there?

Reply:
But also, see my comments below: `hasOnlyOneCredential` may not be needed at all.